src/Builder.php

```diff
@@ -396,6 +397,12 @@ private function buildUrl(string $url, bool $ignoreContentNotFoundErrors = false
         $output = ob_get_clean();
         $content = $response->getContent() ?: $output;
 
+        if ($this->hasNoIndexHeader($response) && !$this->hasNoIndexTag($content)) {
+            $this->logger->warning('Url "{url}" contains a "x-robots-tag: noindex" header that will be lost by going static. Consider using the HTML tag "<meta name="robots" content="noindex" />" instead.', [
+                    'url' => $url,
+                ]);
```

Comment:
Just nitpicking the indent:

```suggestion
            $this->logger->warning('Url "{url}" contains a "x-robots-tag: noindex" header that will be lost by going static. Consider using the HTML tag "<meta name="robots" content="noindex" />" instead.', [
                'url' => $url,
            ]);
```